sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -14,32 +14,67 @@
 package io.dapr.workflows.client;
 
 import com.microsoft.durabletask.DurableTaskClient;
+import com.microsoft.durabletask.DurableTaskGrpcClientBuilder;
+import io.dapr.config.Properties;
+import io.dapr.utils.Version;
+import io.grpc.ManagedChannelBuilder;
 
-public class DaprWorkflowClient {
+import javax.annotation.Nullable;
+
+public class DaprWorkflowClient implements AutoCloseable {
 
   private DurableTaskClient innerClient;
 
   /**
-   * Constructor for DaprWorkflowClient.
+   * Public constructor for DaprWorkflowClient.
+   */
+  public DaprWorkflowClient() throws IllegalArgumentException {
+    this(new DurableTaskGrpcClientBuilder()
```

Comment:
I agree that this is tough to read.  It would also be nice to keep the GRPC_PORT check from the original code.

Can we put this into a private method instead?

`this(createDurableTaskClient());`

`private DurableTaskClient createDurableTaskClient() { ...`